docs/cookbooks/fluentd/README.md

```diff
@@ -101,14 +101,13 @@ kubectl apply -f docs/cookbooks/fluentd/resources/fluentd.yml
 Navigate to `http://[NodeIP]: 30007` and create an index pattern with "fluentd-*".
 Go to `http://[NodeIP]: 30007/app/kibana#/discover` to see the results as below.
 
-<img src="https://downloads.antrea.io/static/10182021/audit-logging-fluentd-kibana.png" width="900" alt="Audit Logging Fluentd Kibana">
+<img src="assets/audit-logging-fluentd-kibana.png" width="900" alt="Audit Logging Fluentd Kibana">
```

Comment:
@antoninbas is it prefered to store image in the website or antrea repo?
BTW, some texts in the new image were not displayed completely, maybe leave some margin above the log record.

Reply:
images should be stored in the website (AWS S3 bucket) to avoid artificially increasing the repo size (especially when images are updated).

Reply:
upload image in the website: https://github.com/antrea-io/antrea/pull/5199/commits/f516a0aa3ddb25bc469fdb467d912acab8d20da6